pages/common/httping.md

```diff
@@ -0,0 +1,19 @@
+# httping
+
+> Measure the latency and throughput of a webserver.
+
+- Ping the webserver on `host`:
+
+`httping -g {{host}}`
+
+- Ping the webserver on `host` and `port`:
+
+`httping -h {{host}} -p {{port}}`
+
+- Ping the webserver on `host` using an SSL connection:
```

Comment:
Perhaps this will read better?

```suggestion
- Ping the web server on `host` using a TLS connection:
```

Reply:
thank you, and I will do better next time :)